lib/cli.js

```diff
@@ -8,57 +8,91 @@ const packageVersion = require("../package").version;
 const jsdiff = require("diff");
 
 const program = new commander.Command();
+const blue = "\x1b[34m";
+const red = "\x1b[31m";
+const withe = "\x1b[37m";
 
 let jsonFile = "";
 let hasErrors = false;
+let fixedFiles = [];
+let wrongFiles = [];
 
 program.
   version(packageVersion,
     "-v, --version",
     "json code style version")
   .option("-f, --fix",
     "fix json file")
+  .option("-q, --quiet",
+    "quiet mode")
   .arguments("[json]")
   .action(function (json) {
     jsonFile = json;
   });
 
 program.parse(process.argv);
 
-function parse(filePath) {
-  let source = fs.readFileSync(filePath, "utf8");
-
-  let formatted = formatter.formatJson(source);
-  let diff = jsdiff.diffJson(source, formatted);
+function parse(path) {
+  if (fs.lstatSync(path).isDirectory()) {
+    readDirectory(path);
+  }
 
-  showDiff(diff);
-  fixJson(filePath, formatted);
+  if (fs.lstatSync(path).isFile()) {
+    processFile(path);
+  }
 
   if (hasErrors) {
+    console.log(wrongFiles);
     process.exit(1);
   }
-
+  
+  !program.fix || console.log(fixedFiles);
   process.exit(0);
 }
 
+function readDirectory(directoryPath) {
```

Comment:
This function is reading files AND processing files. I'd suggest you either rename it or stick to making it just "read" the directory and then get the output with the full file path through the processFile function. Something like:
```js
function readDirectory(dir) {
  ...
  return dir + fileName;
}
...
readDirectory(dirPath).map(processFile); 
```